### PR TITLE
fix: mac arm64 run netlink ut

### PR DESCRIPTION
**Pull request by bobz965**
# Pull Request

- [ ] Make sure you have followed [Kube-OVN Code Style](https://github.com/kubeovn/kube-ovn/blob/master/CODE_STYLE.md).

## What type of this PR

Examples of user facing changes:

- Tests

image

## Which issue(s) this PR fixes

Fixes #(issue-number)

### CODE REVIEW

File: pkg/util/arp_test.go [hunk 1]
```diff
@@ -53,8 +54,11 @@ func TestMacEqual(t *testing.T) {
 }
 
 func TestArpResolve(t *testing.T) {
+	if runtime.GOOS == "darwin" && runtime.GOARCH == "arm64" {
+		t.Skip("Skipping test on macOS ARM64")
+	}
 	// get the default route gw and nic
-	routes, err := netlink.RouteList(nil, netlink.FAMILY_ALL)
+	routes, err := netlink.RouteList(nil, 0)
```
Comment (zhangzujian) [inline pkg/util/arp_test.go:61]:
```suggestion
	routes, err := netlink.RouteList(nil, unix.AF_UNSPEC)
```

File: pkg/util/arp_test.go [hunk 2]
```diff
@@ -53,8 +54,11 @@ func TestMacEqual(t *testing.T) {
 }
 
 func TestArpResolve(t *testing.T) {
+	if runtime.GOOS == "darwin" && runtime.GOARCH == "arm64" {
+		t.Skip("Skipping test on macOS ARM64")
+	}
```
Comment (zhangzujian) [inline pkg/util/arp_test.go:59]:
```suggestion
```

File: pkg/util/arp_test.go (hunk 1 above)
Comment (bobz965) [reply pkg/util/arp_test.go:58]: image
这个用不了，暂时就用0吧

Comment (zhangzujian) [reply pkg/util/arp_test.go:58]: 是 unix 不是 netlink

Comment (bobz965) [reply pkg/util/arp_test.go]: ok 了

**Review by zhangzujian — APPROVED** [review]

**Pull request merged**